docs/Microservices/MPS/configuration.md

```diff
@@ -22,7 +22,7 @@
 | MPS_WEB_ADMIN_PASSWORD     | G@ppm0ym             | Specifies the password for API authentication |
 | MPS_HTTPS                  | true                 | Specifies whether or not to enable https      |
 | MPS_TLS_OFFLOAD            | false                | |
-| MPS_LOG_LEVEL              | info                 | Controls the level of logging provided in the service. Options are (in order of increasing detail): `error`, `warn`, `info`, `verbose`, `debug`  |
+| MPS_LOG_LEVEL              | info                 | Controls the level of logging provided in the service. Options are (in order of increasing detail): `error`, `warn`, `info`, `verbose`, `debug`, and `silly`. |
 | MPS_JWT_SECRET             | supersecret          | Secret used for generating a JWT Token. IMPORTANT: This must match the `secret` in your `Kong.yaml` file for the jwt plugin configuration.
 | MPS_JWT_ISSUER             | 9EmRJTbIiIb4bIeSsmgcWIjrR6HyETqc | The issuer that will be populated in the token. This is a not considered a secret. IMPORTANT: This must match the `key:` property in the `Kong.yaml` file for the jwt plugin configuration.
-| MPS_JWT_EXPIRATION         | 1440                 | The default expiration in minutes for the JWT Token. Default is 24 hours.
\ No newline at end of file
+| MPS_JWT_EXPIRATION         | 1440                 | The default expiration in minutes for the JWT Token. Default is 24 hou
```

Comment:
spelling

Reply:
> spelling

Hmmph. I wonder how that happened. I shall fix it.